Task_Position_the_Cubes/index.css

```diff
@@ -1,27 +1,65 @@
-
-body {
-  background-color: lightblue;
-}
-p{
-  color: darkblue;
-  background-color: yello;
-}
-h1{
-  color: darkblue;
-  /* background-color: hotpink!important; */
+/*  Start coding below for POSITION THE CUBES TASK*/
+.flexbox-container {
+  width: 300px;
+  height: 300px;
+  margin: center; 
+  background-color:blue;
+  flex-grow:3;
+  position: static;
```

Comment:
The class for the second cube should start with '.flexbox-item-2' not '.flexbox-item-3'